.github/workflows/build-upload-artifact.yml

```diff
@@ -61,4 +61,4 @@ jobs:
           name: ${{ steps.get-name.outputs.artifact_name }}
           path: ${{ env.BUILD_DIRECTORY }}
           if-no-files-found: error
-          retention-days: 14
+          retention-days: 14
```

Comment:
Reckon we should remove the retention period from this release as well? Defaults to 90 days.

Reply:
Updated to remove